python/tvm/relay/frontend/tflite.py

```diff
@@ -820,6 +821,38 @@ def convert_transpose(self, op):
 
         return out
 
+    def convert_cast(self, op):
+        """Convert TFLite CAST"""
+        try:
+            from tflite.Operator import Operator
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.CastOptions import CastOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        assert isinstance(op, Operator)
+        input_tensors = self.get_input_tensors(op)
+        assert len(input_tensors) == 1, "input tensors length should be 1"
+        input_tensor = input_tensors[0]
+        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        assert op.BuiltinOptionsType() == BuiltinOptions.CastOptions
+        op_options = op.BuiltinOptions()
+        cast_options = CastOptions()
+        cast_options.Init(op_options.Bytes, op_options.Pos)
+        cast_dtype = cast_options.OutDataType()
+
+        # a dict to decode the enumeriton of tensor dtypes as in the tflite fb schema
+        type_decoder = {
+            0: 'float32',
+            2: 'int32',
+            3: 'uint8',
+            4: 'int64'
+        }
```

Comment:
I was digging for an existing tflite function that to reverse the dtype encoding but could not find so I am open to suggestions

Reply:
We could use `TensorType.UINT8` and so on for `cast_dtype`. I think you could reuse `get_tensor_type_str` function to get string representation of dtype.